libconnman-qt/networkmanager.cpp

```diff
@@ -14,16 +14,16 @@
 #include <QRegularExpression>
 #include <QWeakPointer>
 
-static const QString InputRequestTimeout("InputRequestTimeout");
+static const QString InputRequestTimeoutProperty("InputRequestTimeout");
```

Comment:
Just wondering if it were better or worse if all property names were grouped together...

Reply:
I'll move this a bit.